AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  LiveTranscriptionOptionsViewController.swift
+//  AmazonChimeSDKDemo
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AmazonChimeSDK
+import UIKit
+
+class LiveTranscriptionOptionsViewController: UIViewController {
+    var model: MeetingModel?
+    var meetingID = ""
+    var engineSelected = ""
+    var languageSelected = ""
+    var regionSelected = ""
+    
+    @IBOutlet weak var startTranscriptionButton: UIButton!
+    @IBOutlet weak var engineTextField: UITextField!
+    @IBOutlet weak var languageTextField: UITextField!
+    @IBOutlet weak var regionTextField: UITextField!
+    
+    var engines = ["transcribe_medical", "transcribe"]
+    var languages = ["es-US", "en-US", "en-GB", "en-AU", "fr-CA", "fr-FR", "it-IT", "de-DE", "pt-BR", "ja-JP", "ko-KR", "zh-CN"]
+    var regions = ["us-west-2", "us-east-1", "us-east-2", "sa-east-1", "eu-west-2", "eu-west-1", "eu-central-1", "ca-central-1", "ap-southeast-2", "ap-northeast-2", "ap-northeast-1"]
```

Comment:
Nit: use human friendly names (e.g., ) that map to these values, see: 
https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.html#L335-L340
https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.html#L341-L365
https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.html#L367-L398

Reply:
added more test case information

Reply:
Any reason we still need these separate lists while the dictionary keys already provides enough information?